### PR TITLE
[API] [Order] Adding taxes and promotions to test scope

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: tests/Controller/OrderApiTest.php [hunk 1]
```diff
@@ -81,6 +81,49 @@ public function it_allows_to_get_an_order()
...
+
+        $orderId = $this->prepareOrder();
+
+        $this->client->request('GET', $this->getOrderUrl($orderId), [], [], static::$authorizedHeaderWithAccept);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'order/order_with_promotion_show_response', Response::HTTP_OK);
+    }
+
+
```
Comment (Zales0123) [inline tests/Controller/OrderApiTest.php]: Double blank line.

File: tests/DataFixtures/ORM/resources/checkout.yml [hunk 2]
```diff
@@ -154,3 +157,17 @@ Sylius\Component\Core\Model\Customer:
         email: oliver.queen@star-city.com
         emailCanonical: oliver.queen@star-city.com
         birthday: <date()>
+
+Sylius\Component\Taxation\Model\TaxCategory:
+    regular_tax:
+        code: REGULAR_TAX
+        name: Regular Tax Category
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout.yml]: What would you say for "Taxable Goods"? It's also an example that we use in documentation.

File: tests/DataFixtures/ORM/resources/checkout.yml [hunk 3]
```diff
@@ -154,3 +157,17 @@ Sylius\Component\Core\Model\Customer:
...
+
+Sylius\Component\Core\Model\TaxRate:
+    eu_regular_tax_rate:
+        code: eu_regular_tax_rate
+        name: EU Regular Tax 20%
+        amount: 0.2
+        zone: "@zone_eu"
+        calculator: 'default'
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout.yml]: Why you use apostrophes here if you don't use them anywhere else?

File: tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    holiday_promotion:
+        code: Holliday
+        name: Holliday promotion
+        couponBased: true
+        coupons: ["@some_coupon"]
+        channels: ["@channel"]
+        actions: ["@10_percente_order_discount"]
+
+Sylius\Component\Promotion\Model\PromotionAction:
+    10_percente_order_discount:
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml]: `10_percent_order_discount`

File: tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Sylius\Component\Core\Model\Promotion:
+    holiday_promotion:
+        code: Holliday
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml]: Why not `HOLLIDAY_PROMOTION` or `HOLLIDAY`?

File: tests/Responses/Expected/order/order_with_promotion_show_response.json [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+                "order": {
+                    "href": "@string@"
+                }
+            }
+        }
+    ],
+    "currencyCode": "EUR",
+    "localeCode": "en_US",
+    "checkoutState": "completed"
+}
```
Comment (Zales0123) [inline tests/Responses/Expected/order/order_with_promotion_show_response.json]: Missing blank line

File: tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml (hunk 5 above)
Comment (venyii) [inline tests/DataFixtures/ORM/resources/checkout_coupon_based_promotion.yml]: Ho~~l~~liday

File: tests/DataFixtures/ORM/resources/checkout_promotion.yml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Sylius\Component\Core\Model\Promotion:
+    holiday_promotion:
+        code: Holliday
```
Comment (TheMadeleine) [inline tests/DataFixtures/ORM/resources/checkout_promotion.yml]: Holiday

File: tests/DataFixtures/ORM/resources/checkout_promotion.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+        name: Holliday promotion
```
Comment (TheMadeleine) [inline tests/DataFixtures/ORM/resources/checkout_promotion.yml]: Holiday promotion

File: tests/Responses/Expected/order/order_with_promotion_show_response.json (hunk 6 above)
Comment (lchrusciel) [inline tests/Responses/Expected/order/order_with_promotion_show_response.json]: Added 👍

**Comment by pjedrzejewski** [comment]
@lchrusciel Looks good, but please rebase! :)

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**